react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: 0,
+      switch: false,
+      src: '../images/play.png'
+    };
+    this.clock = this.clock.bind(this);
+    this.reset = this.reset.bind(this);
+  }
+
+  clock() {
+    if (!this.state.switch) {
+      this.setState({
+        switch: true,
+        src: '../images/pause.png'
```

Comment:
Since the image src is decided based on switch, don't put it in state. Instead, choose the src in your render() method.